share/completions/apm.fish

```diff
@@ -61,7 +61,7 @@ end
 
 # Lists all apm config items
 function __fish_apm_config_items
-    apm config list | grep -v '^;\|^$' | tr "\ =\ " "\t"
+    apm config list | string replace -r "\s*=\s*" "=" | string match -r -v '^;|^$' | tr "=" "\t"
```

Comment:
Why not just do the unmatch and then `string replace -r '\s*=\s*' '\t'`? I.e. what's the `tr` still there for?